src/services/dom/parser.js

```diff
@@ -87,8 +87,12 @@ export class Parser {
     const contentType: string = response.headers?.get(
       HTTP_HEADERS.CONTENT_TYPE,
     );
+    const staleHeader: string = response.headers?.get(
```

Comment:
To future-proof this code once we introduce `stale-white-revalidate`, we shouldn't just check for the presence of the header, but also for the value. https://github.com/Instawork/mobile/pull/4697/files#diff-0ff31a656ba253b8f63cd07921f9daf4606bce49c587ccbf5f2b877765af55baR82

Reply:
addressed in https://github.com/Instawork/hyperview/pull/484/commits/664aefe39ad93cc1f0085b0e14ab5175a65e39bb